### ISSUE TITLE
Linefeed in footnote interpreted as end of footnote

### ISSUE THREAD

**Issue by marc-shapiro** [MacDownApp/macdown#982]
A single linefeed is generally interpreted the same as a space.  However, inside a footnote, it is interpreted as the end of the footnote.

MWE follows.
-----

MWE demonstrating footnote issue.[^1]

[^1]: This footnote has two sentences written on the same line.  As expected, both sentences are formatted in the footnote.

Now let's continue the main text.[^2]

[^2]: In this footnote, the following sentence starts on a separate line.
Unfortunately, the continuation sentence is broken away from the footnote.

And this is the end of the main text.

**Comment by JetForMe** [comment]
I have this problem, too.

**Comment by FranklinYu** [comment]
This is confirmed as upstream issue. To avoid it before Hoedown fix it, indent second line like this:

```markdown
This is paragraph.[^1]

[^1]: This is first line.
    This is a following line.
```

**Comment by JetForMe** [comment]
Hmm, that doesn't seem to work for me. If I indent with a tab, it treats it as code. If I indent with four spaces without a  blank line in between, it just runs the two paragraphs together. (this is with 0.7.1).

**Comment by FranklinYu** [comment]
> it just runs the two paragraphs together

This is the behavior of many other Markdown parsers.

**Comment by JetForMe** [comment]
Ah, that may be the case,  but it sure would be nice to support multiple paragraphs. I filed an issue with Hoedown. Thanks!

**Comment by FranklinYu** [comment]
To support multiple paragraph just

```markdown
This is paragraph.[^1]

[^1]: This is first paragraph.

    This is a following paragraph.
```

Besure to test on other implementation, though.

**Comment by JetForMe** [comment]
Yeah, I tried that, too. It just becomes one paragraph. No worries.

**Comment by FranklinYu** [comment]
@JetForMe It’s two paragraphs in my machine. Do you have screenshot?

**Comment by JetForMe** [comment]
Wow, it works now. That was strange. I tried several variants before and it wasn't doing what I wanted. Now it works on both 0.7.1 and master. Sorry for the noise. Thanks!
